nixos/tests/fancontrol.nix

```diff
@@ -0,0 +1,25 @@
+import ./make-test.nix ({ pkgs, ... } : {
+  name = "fancontrol";
+
+  machine =
+    { ... }:
+    { hardware.fancontrol.enable = true;
+      hardware.fancontrol.config = ''
```

Comment:
Since this is only a dummy and the test only tests if the `fancontrol` scripts can be executed one could even make the config shorter?

Reply:
this seems to be based directly on the example in the manual, i'm not sure how one would make it smaller (the given config is minimal)

Reply:
FYI, no, the fancontrol script won't go through any useful/greppable log output without these dummy lines.